src/async_fuse/fuse/fuse_reply.rs

```diff
@@ -731,6 +739,60 @@ impl ReplyXAttr {
     }
 }
 
+/// Fuse delete notification
+#[cfg(feature = "abi-7-18")]
+#[derive(Debug)]
+pub struct FuseDeleteNotification {
+    /// The FUSE device fd
+    fd: RawFd,
+}
+
+#[cfg(feature = "abi-7-18")]
+impl FuseDeleteNotification {
+    /// Create `FuseDeleteNotification`
+    #[must_use]
+    pub const fn new(fd: RawFd) -> Self {
+        Self { fd }
+    }
+
+    /// Notify kernel
+    pub async fn notify(&self, parent: u64, child: u64, name: String) -> nix::Result<usize> {
```

Comment:
Can we reuse the raw reply? Once the message implement AsIoVecList trait it can be send via reply method.

Reply:
notification use error as notification as notification type. This is a little different from raw reply. 
I can add a parameter to raw reply to allow pass-in error code.

Reply:
Done